lib/route/map_route.dart

```diff
@@ -65,6 +69,13 @@ final GoRouter appRoute = GoRouter(
     GoRoute(
       path: onboardingRoute,
       builder: (context, state) => OnboardingScreen(),
-    )
+    ),
+    GoRoute(
```

Comment:
let's do a transition here for the screen to slide up from bottom of the screen. If you need sample code, let me know.